Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Headgear.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73Cap =========== -->	
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/statBases</xpath>
+				<value>
+					<Bulk>1</Bulk>
+					<WornBulk>0.75</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.35</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.08</ArmorRating_Blunt>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/costList</xpath>
+				<value>
+					<costList>
+						<Steel>5</Steel>
+						<Cloth>40</Cloth>
+					</costList>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/apparel/bodyPartGroups</xpath>
+				<value>
+					<bodyPartGroups>
+						<li>UpperHead</li>
+					</bodyPartGroups>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_M73Cap"]/apparel/useDeflectMetalEffect</xpath>
+			</li>
+
+			<!-- ========== SSh40Helmet =========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_SSh40Helmet"]/statBases</xpath>
+				<value>
+					<Bulk>3</Bulk>
+					<WornBulk>1</WornBulk>
```

Comment:
Because the SSh-40 helmet _is_ made of "stuff", we don't need to specify blunt or sharp armor values--it will find them using:

<ArmorRating_ARMORTYPE> = <StuffEffectMultiplierArmor> * <StuffPower_Armor_ARMORTYPE>

Reply:
Fixed in 4378d57